app/src/main/java/sdp/moneyrun/Riddle.java

```diff
@@ -41,9 +46,53 @@ public String getAnswer() {
         return correctAnswer;
     }
 
+
+
+
+    // For some reason this method makes the DB
     public String[] getPossibleAnswers() {
         return new String[]{firstAnswer, secondAnswer, thirdAnswer, fourthAnswer};
     }
 
+    //////Needed for The DataBase Code
+    public String getCorrectAnswer() {
```

Comment:
I don't think all of these getters are needed, but if they are, they should be tested

Reply:
They were there for the Database code, removed now